apiserver/params/storage.go

```diff
@@ -33,3 +33,28 @@ type StorageShowResult struct {
 type StorageListResult struct {
 	Instances []StorageInstance
 }
+
+// PoolInstance holds data for a pool instance.
+type PoolInstance struct {
+	Name   string
+	Type   string
+	Traits map[string]interface{}
+}
+
+// PoolListFilter holds a filter for pool list API call.
+type PoolListFilter struct {
+	Names []string
```

Comment:
Can you comment these please? What are the fields used for? Is there any relationship between Names and Types (i.e. are they pairs?)